scripts/compare-globalvax.py

```diff
@@ -0,0 +1,165 @@
+""" Script that runs through validated manual screenshots in Airtable and uploads them to S3. """
```

Comment:
let's change the header to reflect what we're doing here, like "Script that compares WHO and OWID total vaccine doses and uploads a file with results to S3."